internal/command/init.go

```diff
@@ -159,9 +164,10 @@ func (c *InitCommand) Run(args []string) int {
 			ShowLocalPaths: false, // since they are in a weird location for init
 		}
 
-		ctx, span := tracer.Start(ctx, "-from-module=...", trace.WithAttributes(
-			attribute.String("module_source", src),
+		ctx, span := tracing.Tracer().Start(ctx, "opentofu.init.from_module", trace.WithAttributes(
+			attribute.String("opentofu.module_source", src),
 		))
+		defer span.End()
```

Comment:
This call here should be removed. Check that `span.End()` is called a little bit down the road.